ghci-wrapper/src/Language/Haskell/GhciWrapper.hs

```diff
@@ -142,5 +142,5 @@ evalIt repl expr = do
 -- | Evaluate an expression
 evalEcho :: Interpreter -> String -> IO String
 evalEcho repl expr = do
-  putExpression repl True expr
+  putExpression repl False expr
```

Comment:
@quasicomputational `evalEcho` isn't used by `doctest` + I need this change for `sol/sensei`, so I hope this will be fine.